form/src/App.js

```diff
@@ -29,26 +39,26 @@ function App() {
     showModal();
   };
 
-  const CustomSelect = function (props) {
+  // const CustomSelect = function (props) {
     
-    return (
-      <Select
-        id="input"
-        className="basic-single"
-        classNamePrefix="select"
-        options={props.options.enumOptions}
-        placeholder={props.label}
-        isSearchable={true}
-        onChange={async (e) => {
-          await props.onChange(e.value);
-        }}
-      ></Select>
-    );
-  };
+  //   return (
+  //     <Select
+  //       id="input"
+  //       className="basic-single"
+  //       classNamePrefix="select"
+  //       options={props.options.enumOptions}
+  //       placeholder={props.label}
+  //       isSearchable={true}
+  //       onChange={async (e) => {
+  //         await props.onChange(e.value);
+  //       }}
+  //     ></Select>
+  //   );
+  // };
```

Comment:
Please see your other PR on how to deal with this. 1. have it in a different commit/PR and second don't *comment* code but remove it, it stays in the git log anyway.